src/libstd/path.rs

```diff
@@ -2244,6 +2252,15 @@ impl Path {
     /// assert_eq!(Path::new("./is_a_directory/").is_file(), false);
     /// assert_eq!(Path::new("a_file.txt").is_file(), true);
     /// ```
+    ///
+    /// # See Also
+    ///
+    /// This is a convenience function that ignores errors. If you want to check errors,
```

Comment:
"ignores errors" doesn't seem quite right: the function will return `false` on errors, not ignore them.

I think API functions should be treated as black boxes and `fs::metadata` not so explicitly tied in here. It should just be mentioned in the `See also`.